### PR TITLE
Import 99 problems

### PR DESCRIPTION
**Pull request by tmattio**
This imports the 99 problems data.

The problems are now separated into several files, where each file follows this structure:

```markdown
---
title: ...
number: "..."
difficulty: ...
tags: [ ... ]
---

# Solution

<!-- The solution body here -->

# Statement

<!-- The statement body here -->
```

The Solution and Statement heading must be the only level-one heading in the document.

The given structure gives some more flexibility on how to use the data, but breaks some assumptions from the old data:

- The code defined in previous exercises is not available anymore.
- We can only define one solution per problem

The problems can be rendered now, but not all of them can be compiled with mdx yet, so there's still some work to do before we can merge.

image

### CODE REVIEW

**Comment by patricoferris** [comment]
This looks so nice, thanks @tmattio 🚀

**Comment by patricoferris** [comment]
Maybe we should drop highlight.js into the preview app just to get some colour on all of the code blocks across the previews?

**Comment by tmattio** [comment]
Good idea, done :)

File: data/problems/en/012_run_length_decoding.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+title: Decode a run-length encoded list
+number: "12"
+difficulty: intermediate
+tags: [ "list" ]
+---
+
+# Solution
+
+```ocaml
+# let decode list =
```
Comment (patricoferris) [inline data/problems/en/012_run_length_decoding.md:11]: When this was all one file it probably inherited the types from previous questions and solutions, now they are separate we will have to redefine them

File: data/problems/en/022_create_list.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      if a < b then aux [] b a else List.rev (aux [] a b)
+```
+
+# Statement
+
+If first argument is greater than second, produce a list in decreasing
+order.
+
+```ocaml
+# range 4 9;;
```
Comment (patricoferris) [inline data/problems/en/022_create_list.md:34]: I feel like this should be outputting something right? Like:

```ocaml
- : int list = [4; 5; 6; 7; 8; 9]
```

**Review by patricoferris — COMMENTED** [review]
Started going through this, looks like quite a few code blocks haven't promoted their result, is this intentional ?

**Comment by avsm** [comment]
What is the status of this PR before archiving this repository?

**Comment by tmattio** [comment]
It's not in a mergeable state. I'll work on porting it to the new repo, we can close it in the meantime.

**Pull request closed without merging**